Feature/IN-105: Documents Library

johnyu95 opened the pull request.

johnyu95 committed:
Starting documents page

johnyu95 committed:
Merged with develop

johnyu95 committed:
Added parsley validation to upload form

johnyu95 committed:
Merge branch 'develop' into feature/IN-105-2

* develop:
  Added styling for close button on use policy

johnyu95 committed:
Working on upload view function

johnyu95 committed:
Working on frontend for documents

johnyu95 committed:
Finished styling documents and added pagination

johnyu95 committed:
Merge branch 'develop' into feature/IN-105-2

* develop:
  Capitalize all CTRL-ALT-DEL
  Fixed typo in faq

joelbcastillo committed:
Add virus scan capability; Need to add instructions for installing virus scan utility

joelbcastillo committed:
Add documentation for Virus Scanning

johnyu95 committed:
Added filtering without pagination

johnyu95 committed:
Merged with virus scanning

johnyu95 committed:
Added local timestamps for document uploads

johnyu95 committed:
Finished filtering and pagination for instructions

johnyu95 committed:
Merge branch 'develop' into feature/IN-105-2

* develop:
  Added sponsor to view post page for events

johnyu95 committed:
Created endpoint just for pagination

johnyu95 committed:
Finished everything except pagination for other types

johnyu95 committed:
Finished all pagination types

johnyu95 committed:
Added documentation

johnyu95 committed:
More documentation

joelbcastillo commented:
This pull request **introduces 2 alerts** when merging 15b19a173509d28c77fa03490af9f6ba626ca589 into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-366bfad225a142d324630a97115764fe0eb0c880)

**new alerts:**

* 2 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

joelbcastillo committed:
Updates to Document Upload

joelbcastillo reviewed (approved):
LGTM. Take a look at the changes I just added to make sure you are ok with them.

joelbcastillo committed:
Remove unused import

joelbcastillo committed:
Remove duplicate import in models

joelbcastillo committed:
Configure LGTM to ignore plugins

joelbcastillo committed:
Add FILE_UPLOAD_PATH for Heroku

joelbcastillo commented:
This pull request **fixes 1 alert** when merging 7d34959865b23093c0aa36d533c7c045c9396eb8 into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-2d0f6d0650764fc618e28b6403cf3123dbd7d3f4)

**fixed alerts:**

* 1 for Module is imported more than once

---

*Comment posted by [LGTM.com](https://lgtm.com)*

joelbcastillo committed:
Update Queries for Documents

Now takes deleted files into consideration and returns proper max
based on original query (maximum of results that match the filters
provided by the user)

joelbcastillo commented:
This pull request **fixes 1 alert** when merging 7e04958a92b27f9ec25ace45796497e7f33fa574 into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-7f830968af12e2b3aaca7b71b2f8fa09902b47a1)

**fixed alerts:**

* 1 for Module is imported more than once

---

*Comment posted by [LGTM.com](https://lgtm.com)*

johnyu95 committed:
Fixed parentheses placement for search term

joelbcastillo committed:
Fix query structure for Boolean; Add Documents to manage.py

joelbcastillo committed:
Merge branch 'feature/IN-105-2' of github.com:nycrecords/intranet into feature/IN-105-2

* 'feature/IN-105-2' of github.com:nycrecords/intranet:
  Fixed parentheses placement for search term

johnyu95 committed:
Code cleanup

joelbcastillo commented:
This pull request **introduces 1 alert** and **fixes 1** when merging 53eca3884ab315950b778d90805fb4ddeaeb43ab into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-d0e15edb18bf19df36a2cdc52c516cf382f74614)

**new alerts:**

* 1 for Unused import

**fixed alerts:**

* 1 for Module is imported more than once

---

*Comment posted by [LGTM.com](https://lgtm.com)*

johnyu95 committed:
Shortened queries to always use search term

joelbcastillo commented:
This pull request **introduces 1 alert** and **fixes 1** when merging c007445d60328ba4914a4cfe8ebd7e2e35ecbb5d into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-e04fc4edc2884f84068ea487bd346c3b4dc3d9f4)

**new alerts:**

* 1 for Unused import

**fixed alerts:**

* 1 for Module is imported more than once

---

*Comment posted by [LGTM.com](https://lgtm.com)*

joelbcastillo committed:
Auto load documents in shell

joelbcastillo committed:
Merge branch 'feature/IN-105-2' of github.com:nycrecords/intranet into feature/IN-105-2

* 'feature/IN-105-2' of github.com:nycrecords/intranet:
  Shortened queries to always use search term

joelbcastillo commented:
This pull request **fixes 1 alert** when merging 8ddfed4436163bf86990a1cdac49f71eef38d2d0 into bee7393c77ee2aa19aa26ed1312a80e6c0b1315f - [view on LGTM.com](https://lgtm.com/projects/g/nycrecords/intranet/rev/pr-9b8fe10673d1b14fc14ca6b946480f9ee6d261e6)

**fixed alerts:**

* 1 for Module is imported more than once

---

*Comment posted by [LGTM.com](https://lgtm.com)*
